src/openjd/sessions/_tempdir.py

```diff
@@ -37,7 +37,8 @@ def custom_gettempdir(logger: Optional[LoggerAdapter] = None) -> str:
             program_data_path = r"C:\ProgramData"
             if logger:
                 logger.warning(
-                    f'"PROGRAMDATA" is not set. Set the root directory to the {program_data_path}'
+                    f'"PROGRAMDATA" is not set. Set the root directory to the {program_data_path}',
```

Comment:
```suggestion
                    f'"PROGRAMDATA" is not set. Putting session working directories in {program_data_path} ',
```

Just noticing that this message sounds like instructions to set the dir to C:\ProgramData, but that's not the intent.

Reply:
Modified a bit. New message is:

```'Environment variable "PROGRAMDATA" is not set. Creating the session working directories under C:\ProgramData```